connect/runtime/src/main/java/org/apache/kafka/connect/runtime/isolation/Plugins.java

```diff
@@ -61,12 +61,24 @@ public Plugins(Map<String, String> props) {
         delegatingLoader.initLoaders();
     }
 
+    public Plugins(Map<String, String> props, ClassLoader parent) {
```

Comment:
Is this primarily added for testing purpose?

Reply:
Yep. Added the `VisibleForTesting` comment.

Reply:
I assume then it doesn't have to be `public`, right?

Reply:
Good point. Made the constructor package private.